internal/nginx-meshctl/commands/root.go

```diff
@@ -214,19 +218,19 @@ Will contact Mesh API Server for version and timeout if unable to connect.`,
 	return cmd
 }
 
-func getComponentVersions(k8sClient k8s.Client) (string, error) {
-	// contact mesh-api to get component versions
-	client, err := mesh.NewMeshClient(k8sClient.Config(), meshTimeout)
+// contact mesh controller to get component versions.
+func getComponentVersions(config *rest.Config, namespace string, timeout time.Duration) (string, error) {
```

Comment:
Is it worth writing a unit test for this?

Reply:
We essentially already unit test it on the server side.